pico/builder/tests/builder/src/test/java/io/helidon/pico/builder/api/test/UsageTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.pico.builder.api.test;
+
+import io.helidon.pico.builder.test.testsubjects.A;
+import io.helidon.pico.builder.test.testsubjects.B;
+import io.helidon.pico.builder.test.testsubjects.C;
+import io.helidon.pico.builder.test.testsubjects.DefaultA;
+import io.helidon.pico.builder.test.testsubjects.DefaultB;
+import io.helidon.pico.builder.test.testsubjects.DefaultC;
+import io.helidon.pico.builder.test.testsubjects.DefaultT;
+import io.helidon.pico.builder.test.testsubjects.T;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class UsageTest {
+
+    @Test
+    void testIt() {
+        A a = DefaultA.builder()
+                .a("hello")
+                .build();
+        assertThat(a, notNullValue());
```

Comment:
I would expect assertions on getters here, to make sure they return the configured/default values

Reply:
done; will be in next push.